tcms/tests/__init__.py

```diff
@@ -229,13 +229,15 @@ def setUpTestData(cls):
         cls.execution_6 = executions[5]
 
 
-class AssignPermissionsToTestUser(LoggedInTestCase):
-    """Assign required permissions to user """
+class PermissionsTestCase(LoggedInTestCase):
```

Comment:
when you do a rename also change the other files which are using this class or if you don't intend to update the other test then delete it. The goal is to have pull requests which are always passing CI so they can be merged whenever we decide.

Reply:
I thought my IDE is going to be smart enough and fix it for me.